plugins/com.redhat.fabric8analytics.lsp.eclipse.core/src/com/redhat/fabric8analytics/lsp/eclipse/core/RecommenderAPIProvider.java

```diff
@@ -40,26 +37,25 @@
  */
 public class RecommenderAPIProvider {
 
-	public  static String recommender_api_base_url;
+	public String recommender_api_base_url;
 
 	private static final String RECOMMENDER_API_URL_POSTFIX = "/api/v1";
 
-	private static String userScaleKey;
+	private String userScaleKey;
 
 	private static final String RECOMMENDER_API_URL_STACK_ANALYSES_POSTFIX = RECOMMENDER_API_URL_POSTFIX + "/analyse/";
 	
 	private static final String RECOMMENDER_API_URL_POLL_ANALYSES_POSTFIX = RECOMMENDER_API_URL_POSTFIX + "/stack-analyses/";
 
-	private static final String ANALYSES_REPORT_URL =  "https://stack-analytics-report.openshift.io/#/analyze/";
+	private static final String ANALYSES_REPORT_URL = "https://stack-analytics-report.openshift.io/#/analyze/";
 
-	private static final String POST_ANALYSES_REPORT_URL	= "?api_data={\"access_token\":\"%s\",\"route_config\":{\"api_url\":\"%s\"},\"user_key\":\"%s\"}";
+	private static final String POST_ANALYSES_REPORT_URL = "?api_data={\"access_token\":\"%s\",\"route_config\":{\"api_url\":\"%s\", \"user_key\":\"%s\"},\"show_modal\":false}";
```

Comment:
This will update the code back to using old route which doesn't work properly. New one is ```"?api_data={\"access_token\":\"%s\",\"route_config\":{\"api_url\":\"%s\"},\"user_key\":\"%s\"}";```

Reply:
fixed, thanks! :)